src/module/hooks/index.ts

```diff
@@ -4,4 +4,5 @@ import './ready.ts';
 import './renderChatMessage.ts';
 import './renderItemSheet.ts';
 import './renderSettingsConfig.ts';
+import './updateHits.ts';
```

Comment:
So here I departed from having two files, one for each hook, and instead opted for using more of a "module". But if we want to have one file per hook I can make it so instead.

Reply:
Not sure how I feel about this. One one hand, yeah, a module makes sense. On another, I like being able to easily find the hooks. But, for now at least, sure.